src/paperless/settings.py

```diff
@@ -248,6 +251,9 @@ def deprecated(before, after):
         deprecated("PAPERLESS_CONVERT", "PAPERLESS_CONVERT_BINARY")
         CONVERT_BINARY = os.getenv("PAPERLESS_CONVERT", CONVERT_BINARY)
 
+if not UNPAPER_BINARY:
```

Comment:
It's not, I just thought that was the style you wanted to go for. I'll change it to use the default-param!